src/workloads/execution/ProjectSort.yml

```diff
@@ -0,0 +1,137 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Check performance of a project followed by a sort that can be swapped in order to push down. Check the 
+  performance both without and without an index.
+  Important Metrics: timers.dur, timers.total on the aggregate commands. The index build is to test the performance
+  of the aggregation with/without an index.
+Keywords:
+  indexes
+  collections
+  aggregation
+  sort
+
+GlobalDefaults:
+  aggregate: &collName Collection0
+  pipeline: &aggPipe [{$project: {array: 0}}, {$sort: {integer: 1}}]
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 100000
```

Comment:
You could use the same YAML variable for DocumentCount and BatchSize in order to make sure the mismatch doesn't happen if one of those numbers gets modified.